core/src/main/scala/blueprint/Render.scala

```diff
@@ -72,7 +72,8 @@ object Render {
       EnvironmentVariable("NELSON_ENV", ns.root.asString),
       EnvironmentVariable("NELSON_NAMESPACE", ns.asString),
       EnvironmentVariable("NELSON_PLAN", p.name),
-      EnvironmentVariable("NELSON_STACKNAME", sn.toString)
+      EnvironmentVariable("NELSON_STACKNAME", sn.toString),
+      EnvironmentVariable("NELSON_UNIT", sn.serviceType)
```

Comment:
Sorry to nit-pick but can this be `NELSON_UNITNAME` to be consistent with `STACKNAME` ?